res/css/views/elements/_SettingsFlag.pcss

```diff
@@ -20,6 +20,7 @@ limitations under the License.
     align-items: flex-start;
     justify-content: space-between;
     margin-bottom: 4px;
+    width: 100%;
```

Comment:
Otherwise:

![1](https://github.com/matrix-org/matrix-react-sdk/assets/3362943/7fc50611-1537-404a-9d36-df006274d1ec)

The switches align on `PreferencesTab` without the declaration, though.

![1](https://github.com/matrix-org/matrix-react-sdk/assets/3362943/0374c28b-f1c5-4144-aa44-5d1fabc65f15)

So, I am wondering why the declaration is not required for https://github.com/matrix-org/matrix-react-sdk/pull/10794.

Reply:
ok, but this wasn't needed before (was it?): why is it needed now?

Reply:
Adding `justify-items: stretch` to `mx_SettingsSubsection_content` instead might perhaps be more natural for a grid layout.

![1](https://github.com/matrix-org/matrix-react-sdk/assets/3362943/d9459547-750b-4538-93a2-34e36523f8ba)

Reply:
@kerryarchibald I still don't understand why this is needed now when it wasn't before. I'm not going to hold this review up any further as we've been pinging it back and forth more than long enough and it's somewhat trivial, but if you do get a chance to record a reason here - or better yet, add a comment to the CSS file - then that would be much appreciated.